optimum/intel/openvino/modeling_decoder.py

```diff
@@ -594,7 +595,18 @@ def _from_pretrained(
         causal_model = init_cls(model=model, config=config, model_save_dir=model_cache_path.parent, **kwargs)
 
         if load_in_4bit:
-            compress_decoder_weights(causal_model, quantization_config)
+            if not is_nncf_available():
+                raise ImportError(
+                    "Quantization of the weights requires nncf, please install it with `pip install nncf`"
+                )
+            from .quantization import _weight_only_quantization
+
+            default_config = _check_default_4bit_configs(config)
+
+            if default_config:
+                logger.info(f"For the given model, we recommend the following `quantization_config` : {default_config}")
```

Comment:
I don't think we should overwrite the `quantization_config` wdyt @AlexKoff88

Reply:
Actually, this was an idea behind the combination of `load_in_4bit` + `quantizaiton_config`. If the later one is None we use the pre-defined default config, otherwise we override it an let the user to use a custom config.